lab_2_keywords_cooccurrence/main.py

```diff
@@ -17,7 +17,15 @@ def extract_phrases(text: str) -> Optional[Sequence[str]]:
 
     In case of corrupt input arguments, None is returned
     """
-    pass
+    if not isinstance(text, str) or len(text) == 0:
+        return None
+    delimiters_list = ([',', '!', '?', '\n', ':', ';', ' – ', '¡', '¿', '…', '⋯', '‹', '›', '«', '»', '[', ']',
```

Comment:
Why do you need a `tuple` around a `list`? If you don't change this sequence, use tuple.
Also you can use string - it will take less space

Reply:
+ use some symbols from `string.punctuation`